ReGitLint/Cleanup.cs

```diff
@@ -241,7 +247,7 @@ public override int Run(string[] remainingArguments)
             var runCount = (int)
                 Math.Ceiling(filePaths.Count / (double)BatchSize);
 
-            if (MaxRuns == -1 || runCount > MaxRuns)
+            if (MaxRuns != -1 && runCount > MaxRuns)
```

Comment:
This change looks like introducing a bug. Did you test if it still works properly? Why change it at all?

Reply:
It's intended to fix a bug. If MaxRuns isn't specified it would format all files prior to this change.

Reply:
Tests/FormatCommitA/test.sh was failing before the change.

Reply:
You're right, I initially got it wrong. Thanks for the fix!